code/_onclick/hud/human.dm

```diff
@@ -395,9 +402,9 @@
 	for(var/obj/screen/inventory/inv in (static_inventory + toggleable_inventory))
 		if(inv.slot_id)
 			if(inv.slot_id in S.no_equip)
-				inv.alpha = 128
+				inv.alpha = hud_alpha * 0.5
```

Comment:
`/ 2` can be done here. Divisions and multiplications have mostly the same performance on byond. In some cases divisions are faster than multiplications.
Rule of thumb is to use which one reads the easiest